akka-docs/src/main/paradox/typed/cluster.md

```diff
@@ -95,15 +98,388 @@ Scala
 Java
 :  @@snip [BasicClusterExampleTest.java](/akka-cluster-typed/src/test/java/jdocs/akka/cluster/typed/BasicClusterExampleTest.java) { #cluster-leave-example }
 
-## Serialization
 
-See [serialization](https://doc.akka.io/docs/akka/current/scala/serialization.html) for how messages are sent between
-ActorSystems. Actor references are typically included in the messages,
-since there is no `sender`. To serialize actor references to/from string representation you will use the `ActorRefResolver`.
-For example here's how a serializer could look for the `Ping` and `Pong` messages above:
+### Cluster State
+
+Instead of subscribing to cluster events it can sometimes be convenient to only get the full membership state with
+@scala[`Cluster(system).state`]@java[`Cluster.get(system).state()`]. Note that this state is not necessarily in sync with the events published to a
+cluster subscription.
+
+See [Cluster Membership](cluster-membership.md#member-events) more information on member events specifically.
+There are more types of change events, consult the API documentation
+of classes that extends `akka.cluster.ClusterEvent.ClusterDomainEvent` for details about the events.
+ 
+## Cluster Membership API
+ 
+The `akka.cluster.seed-nodes` are initial contact points for [automatically](#joining-automatically-to-seed-nodes-with-cluster-bootstrap)
+or [manually](#joining-configured-seed-nodes) joining a cluster.
+ 
+After the joining process the seed nodes are not special and they participate in the cluster in exactly the same
+way as other nodes.
+
+### Joining configured seed nodes
+ 
+When a new node is started it sends a message to all seed nodes and then sends a join command to the one that
+answers first. If no one of the seed nodes replied (might not be started yet)
+it retries this procedure until successful or shutdown.
+
+You define the seed nodes in the [configuration](#configuration) file (application.conf):
+
+This can also be defined as Java system properties when starting the JVM using the following syntax:
+
+```
+-Dakka.cluster.seed-nodes.0=akka://ClusterSystem@host1:2552
+-Dakka.cluster.seed-nodes.1=akka://ClusterSystem@host2:2552
+```
```

Comment:
shouldn't this be in the linked configuration section instead?

Reply:
If that were true then much of this page should be :) We have many usage config examples in their respective sections on this page and in the typed usage page. I think it's the simplest way while there are two cluster usage pages going.